app/forms/add_to_order_form.rb

```diff
@@ -25,7 +25,28 @@ def initialize(original_order)
   def save
     raise(ActiveRecord::RecordInvalid, self) unless valid?
 
-    add_to_order!
+    @facility_id = @facility_id&.to_i
+
+    order_project = Projects::Project.find_by(
+      name: "#{current_facility.abbreviation}-#{original_order.id}",
+      facility_id: current_facility.id
+    )
+
+    return add_to_order!(order_project) if @original_order.facility.id == @facility_id
```

Comment:
any reason to keep this separate from the if/else branch below?  I think it could be easier to reason about that way.  for example, what happens when `@original_order.facility.id == @facility_id` is truthy but `order_project.nil?` is also truthy?

Reply:
If we're adding to the original order, we don't care whether there's a project or not. If there is, we just add the new order detail to it. If there isn't, we just create the order detail.